app/utils/callback/tests/data/login-action.json

```diff
@@ -0,0 +1 @@
0.949842\", \"lvl\": \"debug\", \"msg\": \"Calling: 'nice' 'pduclient' '--daemon' 'nuc-2' '--host' 'sain-usb-1' '--port' '15' '--command' 'off'\"}\n- {\"dt\": \"2020-09-10T00:33:32.055874\", \"lvl\": \"debug\", \"msg\": \">> Command sent successfully.\\r\"}\n- {\"dt\": \"2020-09-10T00:33:32.057185\", \"lvl\": \"debug\", \"msg\": \"Returned 0 in 1 seconds\"}\n- {\"dt\": \"2020-09-10T00:33:32.158418\", \"lvl\": \"debug\", \"msg\": \"end: 5.1 power-off (duration 00:00:01) [common]\"}\n- {\"dt\": \"2020-09-10T00:33:32.158775\", \"lvl\": \"results\", \"msg\": {\"case\": \"power-off\", \"definition\": \"lava\", \"duration\": \"1.21\", \"extra\": {\"status\": \"success\"}, \"level\": \"5.1\", \"namespace\": \"common\", \"result\": \"pass\"}}\n- {\"dt\": \"2020-09-10T00:33:32.159178\", \"lvl\": \"debug\", \"msg\": \"start: 5.2 read-feedback (timeout 00:07:31) [common]\"}\n- {\"dt\": \"2020-09-10T00:33:33.161253\", \"lvl\": \"debug\", \"msg\": \"Listened to connection for namespace 'common' for 1s\"}\n- {\"dt\": \"2020-09-10T00:33:33.161591\", \"lvl\": \"info\", \"msg\": \"Finalising connection for namespace 'common'\"}\n- {\"dt\": \"2020-09-10T00:33:33.161837\", \"lvl\": \"target\", \"msg\": \"/ # \"}\n- {\"dt\": \"2020-09-10T00:33:33.262754\", \"lvl\": \"debug\", \"msg\": \"end: 5.2 read-feedback (duration 00:00:01) [common]\"}\n- {\"dt\": \"2020-09-10T00:33:33.263130\", \"lvl\": \"info\", \"msg\": \"end: 5 finalize (duration 00:00:02) [common]\"}\n- {\"dt\": \"2020-09-10T00:33:33.263346\", \"lvl\": \"info\", \"msg\": \"Cleaning after the job\"}\n- {\"dt\": \"2020-09-10T00:33:33.263562\", \"lvl\": \"debug\", \"msg\": \"Cleaning up download directory: /var/lib/lava/dispatcher/tmp/81842/tftp-deploy-rhjmy5up/ramdisk\"}\n- {\"dt\": \"2020-09-10T00:33:33.268133\", \"lvl\": \"debug\", \"msg\": \"Cleaning up download directory: /var/lib/lava/dispatcher/tmp/81842/tftp-deploy-rhjmy5up/kernel\"}\n- {\"dt\": \"2020-09-10T00:33:33.275088\", \"lvl\": \"debug\", \"msg\": \"Cleaning up download directory: /var/lib/lava/dispatcher/tmp/81842/tftp-deploy-rhjmy5up/dtb\"}\n- {\"dt\": \"2020-09-10T00:33:33.275575\", \"lvl\": \"debug\", \"msg\": \"Cleaning up download directory: /var/lib/lava/dispatcher/tmp/81842/tftp-deploy-rhjmy5up/modules\"}\n- {\"dt\": \"2020-09-10T00:33:33.283035\", \"lvl\": \"info\", \"msg\": \"Override tmp directory removed at /var/lib/lava/dispatcher/tmp/81842\"}\n- {\"dt\": \"2020-09-10T00:33:33.352968\", \"lvl\": \"info\", \"msg\": \"Root tmp directory removed at /var/lib/lava/dispatcher/tmp/81842\"}\n- {\"dt\": \"2020-09-10T00:33:33.353353\", \"lvl\": \"info\", \"msg\": \"Job finished correctly\"}\n- {\"dt\": \"2020-09-10T00:33:33.353551\", \"lvl\": \"results\", \"msg\": {\"case\": \"job\", \"definition\": \"lava\", \"result\": \"pass\"}}\n", "priority": 50, "version": "1.1", "description": "kernelci-staging.kernelci.org-staging-20200910.0-arm64-defconfig-gcc-8-bcm2837-rpi-3-b.dtb-bcm2837-rpi-3-b-baseline", "metadata": [], "failure_comment": null}
```

Comment:
Same thing here.

Also, maybe the data could be boiled down to just the minimum required to run the tests.